generate/struct_test.go

```diff
@@ -375,6 +375,7 @@ func TestGoType(t *testing.T) {
 		array       bool
 		required    bool
 		pointers    bool
+		emptyObject bool
```

Comment:
I would add `fields` as a test field and then put in your sample string field to the existing test entry so it keeps working and let yours be `nil` and thus 0.